index.js

```diff
@@ -6,6 +6,18 @@ const ncp = require('copy-paste');
 const recording = require('./recording');
 const del = require('del');
 
+exports.reduceUI = (state, action) => {

```

Comment:
Why are we exporting this if it isn't used anywhere?

Reply:
@cjoshmartin will use the hyperOrama config for merging the desktop capture and canvas recording.